hypervisor/common/hypercall.c

```diff
@@ -664,42 +664,13 @@ int32_t hcall_write_protect_page(struct vm *vm, uint16_t vmid, uint64_t wp_gpa)
 /**
  *@pre Pointer vm shall point to VM0
  */
-int32_t hcall_remap_pci_msix(struct vm *vm, uint16_t vmid, uint64_t param)
+int32_t hcall_remap_pci_msix(__unused struct vm *vm, __unused uint16_t vmid, __unused uint64_t param)
```

Comment:
I think that this function can be removed.
Instead  we can use the below code in vmcall_vmexit_handler
 case HC_VM_PCI_MSIX_REMAP:
                            ret = 0;
                break;

Otherwise we still leave the function with unused parameter, which is not consistent with that code.

Reply:
OK